Gemfile.lock

```diff
@@ -54,7 +54,7 @@ GEM
     jquery-rails (3.1.2)
       railties (>= 3.0, < 5.0)
       thor (>= 0.14, < 2.0)
-    json (1.8.1)
+    json (1.8.3)
```

Comment:
There's a version incompatibility issues with (json 1.8.1) with my local ruby version (ruby 2.3.0p0 (2015-12-25 revision 53290) [x86_64-darwin13]). I will revert to version 1.8.1 before pull request.